README.md

```diff
@@ -36,9 +36,17 @@ Just create a project, and you’re good to go.
 
 Install it once globally:
 
-```sh
-npm install -g create-react-app
```

Comment:
True, but the README is much more of a first-impression than the User Guide (and the "or"s I don't think are too intrusive).

I'm not sure, I just wasn't a huge fan of how this renders out. Maybe if it didn't do a bulleted list, what about just a single line saying "or if you use Yarn", or similar.

/cc @gaearon